testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -29,6 +29,7 @@
     SUPPORTS_TOPN_PUSHDOWN_WITH_VARCHAR(fallback -> fallback.test(SUPPORTS_TOPN_PUSHDOWN) && fallback.test(SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_INEQUALITY)),
 
     SUPPORTS_AGGREGATION_PUSHDOWN,
+    SUPPORTS_AGGREGATION_PUSHDOWN_AVG_BIGINT(false),
```

Comment:
default to SUPPORTS_AGGREGATION_PUSHDOWN

Reply:
👍

Reply:
Take a look at the latest changes @findepi , as I commented here: https://github.com/trinodb/trino/pull/9857#discussion_r743198535

These changes do not apply anymore.